File: LeapYear/LeapYear.py
# WAP to check whether the given year is a Leap Year
try:
    year = int(input("Enter a valid year: "))
    if (year % 400 == 0) or ((year % 4 == 0) and (year % 100 != 0)):
        print(f"{year} is a Leap Year.")
    else:
        print(f"{year} is not a Leap Year.")
except ValueError:
    print("Invalid input. Please enter a valid year as an integer.")

File: LeapYear/LeapYear.js
//  WAP to check whether the given year is a Leap Year or not

year = parseInt(prompt("Enter a valid year: "))
if (isNaN(year)) {
    console.log("You entered wrong input! Enter a valid year.")
}
else {
    if ((year % 400 == 0) || (year % 4 == 0 && year % 100 != 0)) {
        console.log(`${year} is a Leap Year.`)
    }
    else {
        console.log(`${year} is not a Leap Year.`)
    }
}